packages/client-sdk-web/src/internal/vector-index-data-client.ts

```diff
@@ -201,9 +252,43 @@ export class VectorIndexDataClient implements IVectorIndexDataClient {
                   id: hit.getId(),
                   distance: hit.getDistance(),
                   metadata: hit.getMetadataList().reduce((acc, metadata) => {
-                    acc[metadata.getField()] = metadata.getStringValue();
+                    const field = metadata.getField();
+                    switch (metadata.getValueCase()) {
+                      case vectorindex._Metadata.ValueCase.STRING_VALUE:
```

Comment:
curious if there is a reason you are able to use constants here but you had to use string literals for the earlier case statement

Reply:
That appears to be a grpc-js vs grpc-web thing. grpc-js uses string literals and grpc-web constants. 🤸‍♂️

Reply:
cool. bonus points for defining our own constants for it in the near future but definitely not a blocker.